ledger/participant-integration-api/src/main/scala/platform/indexer/JdbcIndexer.scala

```diff
@@ -66,6 +66,14 @@ object JdbcIndexer {
         lfValueTranslationCache,
       )
 
+    def initialized(resetSchema: Boolean = false)(implicit
```

Comment:
Yes, it would probably. What I think is that this class will go away as soon as we cleaned up mutating, so all niceties/refactorings other than necessary is in vain.
Are you okay cleaning this up in the parallel ingestion cleanup epic?